ambry-commons/src/test/java/com.github.ambry.commons/ByteBufferReadableStreamChannelTest.java

```diff
@@ -47,10 +44,32 @@
   @Test
   public void commonCaseTest()
       throws Exception {
-    String[] digestAlgorithms = {"", "MD5", "SHA-1", "SHA-256"};
-    for (String digestAlgorithm : digestAlgorithms) {
-      readToAWCTest(digestAlgorithm);
+    ByteBuffer content = ByteBuffer.wrap(fillRandomBytes(new byte[1024]));
```

Comment:
This code is not new, it has only been moved - It was in a different function that executed the same code on different digest algorithms. Now that there are no digest algorithms, there was no need for another function.